integrationtest/src/test/java/org/hibernate/ogm/test/integration/infinispan/InfinispanModuleMemberRegistrationIT.java

```diff
@@ -41,6 +41,7 @@ private static PersistenceDescriptor persistenceXml() {
 				.name( "primary" )
 				.provider( "org.hibernate.ogm.jpa.HibernateOgmPersistence" )
 				.clazz( Member.class.getName() ).getOrCreateProperties()
+				.createProperty().name( "jboss.as.jpa.providerModule" ).value( "application" ).up()
```

Comment:
Technically it _would_ still work as we provided the "main" alias. But I want to make sure that the tests are properly testing our slotted modules.

The property is needed to prevent Jipijapa from injecting a dependency to "org.hibernate.ogm:main" .

The value "application" means that the user is bundling its own provider and Jipijapa should not add any:
https://docs.jboss.org/author/display/WFLY10/JPA+Reference+Guide